### ISSUE TITLE
Question: Is this a safe way to handle `scaleType` | scaleType does not update the view

### ISSUE THREAD

**Issue by marqroldan** [blackuy/react-native-twilio-video-webrtc#386]
### Steps to reproduce
- Use a working example and add a variable for `scaleType` to switch between modes

### Expected behaviour
- Switching should update the view

### Actual behaviour
- Switching does not update the view with just the change in `scaleType`, if you change the key, however, it updates properly.

### Environment
- **Node.js version**:  12.18.4
- **React Native version**: 0.63.2
- **React Native platform + platform version**: Android 9.0

### react-native-twilio-video-webrtc
**Version**: master

-----

I can update it if I update the key (currently doing `${trackSid}_${this.state.scaleType}`)
However, does it have any negative impact? Right now I don't think there is, but I'm opening the issue just in case

**Comment by slycoder** [comment]
Yeah, looks like the code is not properly checking and updating the view when the prop changes.
